jalejandromj/react-apprenticeship-capstone-2
# Final deliverable Capstone 2

## Description
```
App that initially displays the "image of the day" corresponding to today (using the NASA APOD API). User has the option to search by a desired date as well. Responsive breakpoints 1280, 1366, and 1920px (can be later adjusted).
```
## Type of Change
Final deliverable
```
Delete options that are not relevant.
```


## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 

- To see tests and coverage, run: "npm run test"
```
## Any background context you want to provide?
```

```
## What is the deliverable?
```
Final deliverable
```
## Screenshots and links
```
https://alejandromj-img-of-day.netlify.app/
```
## Notes
```
My main concern (again) are the tests that were done. Overall, I think they are a little bit more complete, but I was not able to mock axios.get (Home.test) and had some issues testing changing values in Context that would later affect a component (Layout.test, default displaySidebar = true; wanted to test sidebar hide on Context displaySidebar = false)
```
## Checklist
